release/ray_release/byod/byod_increase_ray_max_limit_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# This script is used to build an extra layer on top of the base anyscale/ray image 
+# to run the horovod tests
+
+set -exo pipefail
+
+echo "export RAY_MAX_LIMIT_FROM_API_SERVER=1000000000" >> /home/ray/.bashrc
+echo "export RAY_MAX_LIMIT_FROM_DATA_SOURCE=1000000000" >> /home/ray/.bashrc
```

Comment:
why adding it into `.bashrc` fixes it? yeah, I have the same feeling that this is not the right way.

Reply:
byod runtime_env is set during ray.init; but these two variables need to be set before ray.start (which is also before ray.init)

Reply:
same as the other comment: why not add a env list here under byod, and add ENV RAY_MAX_LIMIT_FROM_API_SERVER=1000000000 into the Dockerfile